src/ixdat/techniques/ec.py

```diff
@@ -198,35 +198,39 @@ def __init__(
         self._raw_current = None
         self._selector = None
         self._file_number = None
-        if all(
-            [
-                (current_name not in self.series_names)
-                for current_name in self.raw_current_names
-            ]
-        ):
-            self.series_list.append(
-                ConstantValue(
-                    name=self.raw_current_names[0],
-                    unit_name="mA",
-                    value=0,
+        if self.potential:
+            if all(
+                [
+                    (current_name not in self.series_names)
+                    for current_name in self.raw_current_names
+                ]
+            ):
+                self.series_list.append(
+                    ConstantValue(
+                        name=self.raw_current_names[0],
+                        unit_name="mA",
+                        value=0,
+                    )
                 )
-            )
-            self._populate_constants()  # So that OCP currents are included as 0.
-            # TODO: I don't like this. ConstantValue was introduced to facilitate
-            #   ixdat's laziness, but I can't find anywhere else to put the call to
-            #   _populate_constants() that can find the right tseries. This is a
-            #   violation of laziness as bad as what it was meant to solve.
-        if all(
-            [(cycle_name not in self.series_names) for cycle_name in self.cycle_names]
-        ):
-            self.series_list.append(
-                ConstantValue(
-                    name=self.cycle_names[0],
-                    unit_name=None,
-                    value=0,
+                self._populate_constants()  # So that OCP currents are included as 0.
+                # TODO: I don't like this. ConstantValue was introduced to facilitate
```

Comment:
Yeah, good point... But then the plotter would have to go through individual ValueSeries, rather than use the synthesized ValueSeries that `grab()` works with. Do you think it should do so?